lib/src/track/options.dart

```diff
@@ -113,6 +113,194 @@ extension VideoEncodingExt on VideoEncoding {
 }
 
 class VideoParameters {
+  // 16:9 Presets
+  static const presetH90_169 = VideoParameters(
+    dimensions: VideoDimensions.h90_169,
+    encoding: VideoEncoding(
+      maxBitrate: 60 * 1000,
+      maxFramerate: 15,
+    ),
+  );
+  static const presetH180_169 = VideoParameters(
+    dimensions: VideoDimensions.h180_169,
+    encoding: VideoEncoding(
+      maxBitrate: 120 * 1000,
+      maxFramerate: 15,
+    ),
+  );
+
+  static const presetH216_169 = VideoParameters(
+    dimensions: VideoDimensions.h216_169,
+    encoding: VideoEncoding(
+      maxBitrate: 180 * 1000,
+      maxFramerate: 15,
+    ),
+  );
+
+  static const presetH360_169 = VideoParameters(
+    dimensions: VideoDimensions.h360_169,
+    encoding: VideoEncoding(
+      maxBitrate: 300 * 1000,
+      maxFramerate: 20,
+    ),
+  );
+
+  static const presetH540_169 = VideoParameters(
+    dimensions: VideoDimensions.h540_169,
+    encoding: VideoEncoding(
+      maxBitrate: 600 * 1000,
+      maxFramerate: 25,
+    ),
+  );
+
+  static const presetH1080_169 = VideoParameters(
+    dimensions: VideoDimensions.h1080_169,
+    encoding: VideoEncoding(
+      maxBitrate: 3 * 1000 * 1000,
+      maxFramerate: 30,
+    ),
+  );
+
+  static const presetH1440_169 = VideoParameters(
+    dimensions: VideoDimensions.h1440_169,
+    encoding: VideoEncoding(
+      maxBitrate: 5 * 1000 * 1000,
+      maxFramerate: 30,
+    ),
+  );
+
+  static const presetH2160_169 = VideoParameters(
+    dimensions: VideoDimensions.h2160_169,
+    encoding: VideoEncoding(
+      maxBitrate: 8 * 1000 * 1000,
+      maxFramerate: 30,
+    ),
+  );
+
+  static const presetH720_169 = VideoParameters(
+    dimensions: VideoDimensions.h540_169,
```

Comment:
Typo? I think this should be h720.. also move to below h540? :)

Reply:
😅🙏🏻